providers/apache/hive/src/airflow/providers/apache/hive/operators/hive.py

```diff
@@ -26,8 +26,7 @@
 from airflow.configuration import conf
 from airflow.models import BaseOperator
 from airflow.providers.apache.hive.hooks.hive import HiveCliHook
-from airflow.utils import operator_helpers
-from airflow.utils.operator_helpers import context_to_airflow_vars
+from airflow.sdk.execution_time.context import AIRFLOW_VAR_NAME_FORMAT_MAPPING, context_to_airflow_vars
```

Comment:
We'll need to import these from [common.compat provider](https://github.com/apache/airflow/blob/main/providers/common/compat/README.rst) for backward compat